File: Introduction to Computer Science/Week 1 - C/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{

    //declare dollars
    float dollars;

    // get input until input is greater than 0
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars <= 0);

    // chage dollars to cents
    int cents = round(dollars * 100);

    // initialize number of coins
    int numOfCoins = 0;


    // loop until cents equal or less than 0
    while (cents > 0)
    {

        // substract conditionally
        if (cents >= 25)
        {
            cents -= 25;
            numOfCoins++;
        }
        else if (cents >= 10)
        {
            cents -= 10;
            numOfCoins++;
        }
        else if (cents >= 5)
        {
            cents -= 5;
            numOfCoins++;
        }
        else
        {
            cents -= 1;
            numOfCoins++;
        }
    }

    // print number of coins
    printf("%i\n", numOfCoins);
}

File: Introduction to Computer Science/Week 6 - Python/cash/cash.py
from cs50 import get_float

cash = get_float("Changed owed: ")

while cash <= 0:
    cash = get_float("Changed owed: ")

cash = round(cash * 100)
coins = 0

while cash > 0:
    if cash >= 25:
        cash -= 25
        coins += 1
    elif cash >= 10:
        cash -= 10
        coins += 1
    elif cash >= 5:
        cash -= 5
        coins += 1
    else:
        cash -= 1
        coins += 1

print(f"{coins}")